pkg/registry/storage/csidriver/strategy_test.go

```diff
@@ -74,6 +78,222 @@ func TestCSIDriverStrategy(t *testing.T) {
 	}
 }
 
+func TestCSIDriverCreate(t *testing.T) {
```

Comment:
Suffix the function names with PrepareForCreate/Update, to reflect that we're only testing those methods

Reply:
Done.